application/modules/g/controllers/AuthController.php

```diff
@@ -448,9 +448,28 @@ public function validateemailAction() {
 					$auth->store($userData, $method);
 				}
 			}
+			$this->view->user = $user;
 		}
 	}
 
+	public function sendvalidateemailAction() {
+		$auth = Garp_Auth::getInstance();
+		if (!$this->getRequest()->isPost() || !$auth->isLoggedIn()) {
+			throw new Zend_Controller_Action_Exception('Page not found', 404);
+		}
+		
+		$userModel = new Model_User();
+		$currentUser = $userModel->fetchById($auth->getUserId());
+		$userModel->invalidateEmailAddress($currentUser, 'update');
+
+		$authVars = $auth->getConfigValues();
+
+		// $flashMessage = __($authVars['validateemail']['validation_email_send_message']);
+
+		// $this->_helper->flashMessenger($flashMessage);
```

Comment:
Hmm, waarom doen we geen flash message hier? Is wel netjes toch?    
Zo niet dan is regel 465 ook overbodig trouwens.